server/src/main/java/de/uftos/services/StudentService.java

```diff
@@ -29,20 +32,31 @@ public class StudentService {
   private final StudentRepository repository;
   private final StudentGroupRepository studentGroupRepository;
   private final ServerRepository serverRepository;
+  private final ConstraintSignatureRepository constraintSignatureRepository;
+  private final ConstraintInstanceRepository constraintInstanceRepository;
```

Comment:
Checkstyle (lexicographical order of imports).

Reply:
done